EpikCord/__init__.py

```diff
@@ -747,9 +745,7 @@ async def send(
             payload["sticker_ids"] = sticker_ids
 
         if attachments:
-            payload["attachments"] = [
-                attachment.to_dict() for attachment in attachments
-            ]
+            logger.error("Attachments for files are NOT supported")
```

Comment:
Attachments != Files

Reply:
that is just a temporary error, will fix it later in this PR
The reason i removed it is that there are no attachment.to_Dict

Reply:
What are they, anyway

Reply:
https://discord.com/developers/docs/resources/channel#attachment-object

Reply:
https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-messages